### DISCUSSION TITLE
월드 1.4.0 버전 Rect Transform 인스팩터로 지정할 수 없는 문제

### DISCUSSION THREAD

**Discussion by iamarohan** [naverz/zepeto-studio-kor#857] [5. Q&A]
안녕하세요!

월드 1.4.0 버전 업데이트 이후 인스펙터에 Drag and drop으로 rect transform을 지정할 수 없는 문제가 발생하는 것 같습니다!
코드를 건드리지 않고 1.3.0으로 바꾸었더니 되는 기능이라 버그인 것 같은데, 확인 부탁드립니다!

감사합니다!

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 버그 제보는 이슈 트래킹 및 관리를 위해 게시판에 부탁 드립니다.
https://github.com/naverz/zepeto-studio-kor/discussions/700
감사합니다.
